MAINTAINERS.md

```diff
@@ -0,0 +1,32 @@
+# Maintainers
+
+This page includes information for maintainers of this repo.
+
+## How to manually generate the binaries for a Github release
```

Comment:
These docs were moved from the README b/c it's really only helpful for maintainers in the case the GH workflow fails.